Wikipedia/UI-V5/WMFTableOfContentsAnimator.swift

```diff
@@ -0,0 +1,59 @@
+
+import UIKit
+
+class WMFTableOfContentsAnimator: NSObject, UIViewControllerAnimatedTransitioning {
+    
+    // MARK: - init
+    let isPresenting :Bool
```

Comment:
any reason why we can't have two animator classes? something else apparently knows when to chose one over the other, and I can't see ay benefit to having both functions in the same class which could also be had w/ a shared base class